lib/compiler/test/compile_SUITE.erl

```diff
@@ -727,6 +727,23 @@ utf8_atoms(Config) when is_list(Config) ->
     NoUtf8AtomForms = [{attribute,Anno,module,no_utf8_atom}|Forms],
     error = compile:forms(NoUtf8AtomForms, [binary, r19]).
 
+utf8_functions(Config) when is_list(Config) ->
+    Anno = erl_anno:new(1),
+    Atom = binary_to_atom(<<"こんにちは"/utf8>>, utf8),
```

Comment:
@c0b This uses the same template as the test above and at the time the parser did not handle UTF-8 one. It is probably best to not be dependent on the parser on this case still.